packages/luna/src/components/StudentOverview/StudentOverview.tsx

```diff
@@ -48,11 +48,13 @@ export const StudentOverview: React.FC<Props> = ({
      * The plan is to use `humanize-duration` below 1k hours then do manual
      * formating beyond this.
      */
+    const duration = Duration.from(totalLearningTime.toString());
+
+    if (totalLearningTime < 60)
+      return duration.hours({ language: "ar-short", units: ["m"] });
+
     if (totalLearningTime < 60 * 1000)
-      return formatMinutes(totalLearningTime, {
-        units: ["h"],
-        maxDecimalPoints: 0,
-      });
+      return duration.hours({ language: "ar-short" });
 
     const hours = Math.floor(totalLearningTime / 60);
     return intl("student-dashboard.overview.total-learning-time.unit", {
```

Comment:
Based on your changes, we should update this logic as well. to use `duration.format`

Reply:
The method named `duration.hours` is misleading. We should use `duration.format` and modifiy to be customizable to accept options.

Reply:
yes I changed the formate function.

I didn't change the logic of line `59` as humanize doesn't formate numbers so i left it as it's